### DISCUSSION TITLE
Using action under MSYS2

### DISCUSSION THREAD

**Discussion by daljit46** [Mozilla-Actions/sccache-action#46] [Q&A] [answered]
Hi, 
I'd like to make use of this action using MSYS2 (following the instructions [here](https://www.msys2.org/docs/ci/)) with gcc. Is this supported or is the usecase specifically catered for using MSVC on Windows? Can I simply pass the path of sccache to the MSYS2 console and then run sccache from there?

**Comment by uulm-janbaudisch** [comment, marked as answer]
Hi, this question somewhat old know, but I thought I might share my solution anyway. sccache as installed with this action seems to work in an MSYS2 environment. It is not available in `$PATH`, so it must be called via `$SCCACHE_PATH`.
For example, to use sccache with CMake on MSYS2, you can use the following steps in your workflow:

```
echo CMAKE_C_COMPILER_LAUNCHER="$SCCACHE_PATH" >> $GITHUB_ENV
echo CMAKE_CXX_COMPILER_LAUNCHER="$SCCACHE_PATH" >> $GITHUB_ENV
```

In steps following this one, CMake would be able to use sccache.
